feat(guide): Recommended naming scheme - Add Enable Rename instructions (#1800)
TRaSH-Guides/Guides: docs/Radarr/Radarr-recommended-naming-scheme.md

![Unhide Advanced](images/unhide-advanced.png)

> And as last enable Rename Movies to reveal the movies renaming options.

Reviewer: ```suggestion
Then, enable Rename Movies to reveal the movies renaming options.
```

> And as last enable Rename Movies to reveal the movies renaming options.

Reviewer 2: ```suggestion
Finally, enable `Rename Movies` to reveal the movie renaming options.
